### ISSUE TITLE
Starting a list of barcoding genes and their synonyms.

### ISSUE THREAD

**Issue by tobiasgf** [gbif/edna-tool-ui#93]
At some point we may want to make a more comprehensive list of barcoding genes and their synonyms (also for use in the sequence ID tool, etc.)

Starting here (with a bit of help from AI):

- **12S rRNA** (12S ribosomal RNA): 12SrDNA, 12S
- **16S rRNA (mitochondrial)** (16S ribosomal RNA): 16SrDNA, 16S
- **16S rRNA (bacterial)** (16S ribosomal RNA)
- **18S rRNA** (18S ribosomal RNA): 18SrDNA, 18S
- **23S rRNA** (23S ribosomal RNA)
- **28S rRNA** (28S ribosomal RNA)
- **rbcL** (Ribulose-bisphosphate carboxylase large chain)
- **CytB** (Cytochrome b): CYTB, CYB
- **COI** (Cytochrome c oxidase I): CO1, COX1
- **COII** (Cytochrome c oxidase II): CO2, COX2
- **COIII** (Cytochrome c oxidase III): CO3, COX3
- **nifH** (Nitrogenase reductase)
- **ITS** (Internal Transcribed Spacer): ITS1, ITS2
- **ND1** (NADH dehydrogenase subunit 1): NAD1
- **ND2** (NADH dehydrogenase subunit 2): NAD2
- **ND3** (NADH dehydrogenase subunit 3): NAD3
- **ND4** (NADH dehydrogenase subunit 4): NAD4
- **ND5** (NADH dehydrogenase subunit 5): NAD5
- **ND6** (NADH dehydrogenase subunit 6): NAD6
- **amoA** (Ammonia monooxygenase subunit A)
- **rpoB** (RNA polymerase subunit B)
- **rpoC1** (RNA polymerase subunit C1)
- **rpoC2** (RNA polymerase subunit C2)
- **matK** (Maturase K)
- **trnH** (tRNA-His)
- **trnL** (tRNA-Leu)
- **psbK** (Photosystem II protein K)
- **trnK** (tRNA-Lys)
- **D-loop**
- **atp6** (ATP synthase F0 subunit 6)
- **atp8** (ATP synthase F0 subunit 8)
- **tufA** (Elongation factor Tu)

**Comment by tobiasgf** [comment]
Consider using rDNA instead of rDNA, reflecting the focus on DNA sequences and not the gene product.

**Comment by tobiasgf** [comment]
Also, consider splitting ITS into: ITS1 and ITS2, and ITS region (ITS1+5.8S+ITS2)

**Comment by CecSve** [comment]
> * **16S rRNA (mitochondrial)** (16S ribosomal RNA): 16SrDNA, 16S
> 
> * **16S rRNA (bacterial)** (16S ribosomal RNA)

Does it matter whether it is mitochondrial or bacterial? Concepts have to be unique, so I have added only one.

**Comment by tobiasgf** [comment]
Two different genes
16S mitochondrial gene is the Large subunit (LSU) (homologue to the 23S in bacteria and 28S in eukaryotes)
16S bacterial gene is the small subunit (SSU) (homologue to the 12S in mitochondria and 28S in eukaryotes)
They just (unfortunately) have the same sedimentation coefficient (S-value).

**Comment by CecSve** [comment]
What do you propose we call the concepts then so we make sure they do not match?

**Comment by tobiasgf** [comment]
Also, I think we should use the DNA-terminology (rDNA not rRNA), as we are talkning about the DNA that is being targeted, not the gene product.

**Comment by CecSve** [comment]
Could I ask you to update the first post with the list? Then I'll add the concepts to the spreadsheet.

**Comment by CecSve** [comment]
> Also, consider splitting ITS into: ITS1 and ITS2, and ITS region (ITS1+5.8S+ITS2)

I have added 1 and 2 - not sure what is meant with the region. Could you please specify? https://docs.google.com/spreadsheets/d/1_cV4LZeqF_sm-JVaHuKusWBeVK2VkVWEcuPM8MPZN3s/edit#gid=1674909017

**Comment by CecSve** [comment]
> Two different genes 16S mitochondrial gene is the Large subunit (LSU) (homologue to the 23S in bacteria and 28S in eukaryotes) 16S bacterial gene is the small subunit (SSU) (homologue to the 12S in mitochondria and 28S in eukaryotes) They just (unfortunately) have the same sedimentation coefficient (S-value).
> 
> So the unique concepts would rather be LSU and SSU. But they are just targeted with very different primers.
> 
> But I believe it is best to keep these two genes (LSU and SSU) separate between bacteria/archaea, eukatyotes and mitochondria.

I do not think it is currently captured well as concepts. Ideally, it should be clear for publishers as well as users - but mostly users what goes in the interpreted field. Let me know how you think we should separate the two in a clear way.

**Comment by CecSve** [comment]
The verbatim values are now ready to be mapped - I have added some mapping already, however, since they all are identical to either the concept, label_en or alternativeLabel_en they will be removed from the mapping sheet since they will be mapped based on this.

**Comment by only1chunts** [comment]
fyi - i have linked a [GSC MIxS ticket](https://github.com/GenomicsStandardsConsortium/mixs/issues/887) to this so that we can update the 'target_gene' accepted values list with the consensus list here, thanks.

**Comment by tobiasgf** [comment]
> fyi - i have linked a [GSC MIxS ticket](https://github.com/GenomicsStandardsConsortium/mixs/issues/887) to this so that we can update the 'target_gene' accepted values list with the consensus list here, thanks.

Very good, @only1chunts. I don't yet consider this list as authoritative or community accepted. This is a first attempt to collect a list of barcode genes (and their synonyms / alternative names).

**Comment by tobiasgf** [comment]
> Also, I think we should use the DNA-terminology (rDNA not rRNA), as we are talkning about the DNA that is being targeted, not the gene product. Maybe:
> 
> 12S mtDNA 16S mtDNA 16S rDNA 18S rDNA 23S rDNA 28S rDNA

Based on more recent speculations and discussions, a suggestion for an unambiguous set of labels for the commonly used ribosomal genes could be.
12S rRNA (SSU mitochondria)
16S rRNA (LSU mitochondria)
16S rRNA (SSU prokaryote)
23S rRNA (LSU prokaryote)
18S rRNA (SSU eukaryote)
28S rRNA (LSU eukaryote) 
(The specification for the eukaryotes may not be needed, as 18S and 28S are unambiguous. But for a systematic labelling, they may be relevant.)

**Comment by cpavloud** [comment]
> Consider using rDNA instead of rDNA, reflecting the focus on DNA sequences and not the gene product. [edit, I meant: rDNA instead of rRNA]

Technically, there is no such thing as rDNA in an actual living organism (despite the fact that you can find it in the literature...), so I would keep the rRNA. 
If you want to be more specific, you can use a sentence like "the gene encoding for 16S rRNA", or "the 16S rRNA gene" or something like that.

**Comment by cpavloud** [comment]
I think the main issue here arises from the primer (non-)specificities.
It is very common that e.g. with 16S rRNA you amplify Eukaryotes when aiming for Bacteria/Archaea.

So, maybe instead of modifying the [target_gene ](https://genomicsstandardsconsortium.github.io/mixs/0000044/) term or proposing alternative values, we should be aiming to create a new **target_organism** term?

**Comment by only1chunts** [comment]
I think a new term is a completely different conversation*, the general
idea of the target gene term is to collect information about which gene was
targeted, not necessarily about what was actually obtained after analysis.
I think the list here is along the right lines.
* - that conversation sounds like the sort of thing the eDNA working group
would having (I can put you in touch with Miwa if you're not already in
that group)

**Comment by cpavloud** [comment]
I am not in the group but I would love to be, so it would be great to put me in touch with the relevant people. 
I don't want to hijack the conversation here.

**Comment by sformel-usgs** [comment]
This conversation moved fast! I was traveling last week, but here are my thoughts.  

1. Creating a controlled vocabulary to help clean up this mess is a great idea, it's sorely needed.
2. To future proof this a bit, I think we should explore chunking this up by cellular compartment using GO terms. This might be too onerous for users, but it also might raise awareness about the diversity of data that needs to be interoperable for data sharing purposes.
   - https://amigo.geneontology.org/amigo/term/GO:0005761
   - https://amigo.geneontology.org/amigo/term/GO:0000315
   - https://amigo.geneontology.org/amigo/term/GO:0000314
   - If needed, you can drill down in the above terms for more specificity (e.g. chloroplast LSU)
3. Since primers may detect off-target taxa, IMO, it would be best if the taxonomy was kept in taxonomy terms, rather than be implied in the gene target terms.
